azure/pipelines/build.yml

```diff
@@ -291,6 +291,7 @@ stages:
       dsiApiSecret: '$(dsiApiSecret)'
       ucasUsername: '$(ucasUsername)'
       ucasPassword: '$(ucasPassword)'
+      ucasZipPassword: '$(ucasZipPassword)'
```

Comment:
I recall @vigneshmsft said entries in build.yml and release.yml are no longer needed when I added ucasUploadBaseUrl etc.

Reply:
Let's fix that in a separate PR, I don't want to introduce inconsistency between different sets of variables, since this stuff is so hard to test.